Makefile

```
@@ -50,6 +50,10 @@ lint-html:  ## Run HTML Linters
 lint-frontend:  ## Run front-end linters
 	npm run lint
 
+.PHONY: lint-migrations
+lint-migrations: ## Run django-migration-linter
+	python manage.py lintmigrations
```

```suggestion
	poetry run python manage.py lintmigrations
```

so it uses the virtual environment with all the installed dependencies. Should fix the `ModuleNotFoundError: No module named 'django'` error